### PR TITLE
[BACKLOG-6897] -- updated OperationAPI#submit to allow for options

### PR DESCRIPTION
**Pull request by krivera-pentaho**

### CODE REVIEW

**Comment by krivera-pentaho** [comment]
@pentaho/orlandocalisbon please review

File: test-js/unit/prompting/api/OperationAPISpec.js [hunk 1]
```diff
@@ -162,10 +162,11 @@ define(["common-ui/prompting/api/OperationAPI"], function(OperationAPI) {
     });
 
     it("should test submit", function() {
-      operationApi.submit();
+      var options = {isInit: true};
+      operationApi.submit(options);
 
       expect(operationApi._getPromptPanel).toHaveBeenCalled();
-      expect(promptPanelSpy.submit).toHaveBeenCalled();
+      expect(promptPanelSpy._submit).toHaveBeenCalledWith(options);
```
Comment (diogofscmariano) [inline test-js/unit/prompting/api/OperationAPISpec.js]: Any reason to create a new _submit? Shouldn't we use the old one?

**Comment by diogofscmariano** [comment]
Apart from the comment with _submit no other issues

**Comment by krivera-pentaho** [comment]
Updated pull request. Please review. @pentaho/orlandocalisbon

**Pull request merged**